evap/staff/templates/staff_user_importer.html

```diff
@@ -0,0 +1,114 @@
+                        <div class="card-header">
+                            <a class="collapse-toggle" data-bs-toggle="collapse"
+                               href="#{{ category_group.grouper.value.id }}Card"
+                               aria-expanded="false"
+                               aria-controls="{{ category_group.grouper.value.id }}Card">
+                                {{ category_group.grouper.value.display_name }}
+                            </a>
+                        </div>
+                        <div class="collapse show" id="{{ category_group.grouper }}Card">
+                            <div class="card-body">
+                                <div class="w-75 container text-center">
+                                    <div class="row row-cols-3">
+                                        <div class="col-lg">{% trans 'Current user' %}</div>
+                                        <div class="col-2">{% trans 'Change?' %}</div>
+                                        <div class="col-lg">{% trans 'Imported user' %}</div>
+                                    </div>
+                                    {% for decision in decision_field %}
+                                        <div class="row row-cols-3 d-flex p-0 form-check form-switch">
+                                            <label class="col-lg form-check-label"
+                                                   for="{{ decision_field.id_for_label }}_{{ forloop.counter }}">
+                                                {{ decision.data.label.existing }}
+                                            </label>
+                                            <div class="col-2">
+                                                <input class="form-check-input mx-auto float-none"
+                                                       id="{{ decision_field.id_for_label }}_{{ forloop.counter }}"
+                                                       name="{{ decision.data.name }}" type="checkbox"
+                                                       value="{{ decision.data.label.imported }}" />
+                                            </div>
+                                            <label class="form-check-label col-lg"
+                                                   for="{{ decision_field.id_for_label }}_{{ forloop.counter }}">
+                                                {{ decision.data.label.imported }}
+                                            </label>
+                                        </div>
+                                    {% endfor %}
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                {% endif %}
+            {% endfor %}
+        {% endfor %}
+        <button type="submit" class="btn btn-primary mb-3">
+            {% trans 'Resolve and download excel' %}
+        </button>
```

Comment:
Currently, the submit button is not next to the `Upload and test`. This may look weird. The problem is, that this is a different form.